pkgs/development/tools/misc/distcc/default.nix

```diff
@@ -29,6 +33,8 @@ let
                             --disable-Werror   # a must on gcc 4.6
                            )
       installFlags="sysconfdir=$out/etc";
+
+      ./autogen.sh
     '';
     patches = [ ./20-minute-io-timeout.patch ];
```

Comment:
IMHO it's fine to drop the timeout patch altogether. Waiting 20 minutes on a network resource doesn't seem like a good idea in this day and age anymore.